balboa-http/src/it/scala/com/socrata/balboa/server/MetricsIntegrationTest.scala

```diff
@@ -219,19 +220,73 @@ class MetricsIntegrationTest extends FlatSpec with Matchers with BeforeAndAfterE
 
     "{" +
       metRange.map(i =>
-        """ "%s-%d": { "value": 1, "type": "absolute" } """
-          .format(testMetricName, i)
+        s""" "$testMetricName-$i": { "value": 1, "type": "${metricType.toString}" } """
       ).mkString(", ") +
       "}"
   }
 
+  def persistManyAbsoluteAndAggregate(): String = {
+    val url = new URL(Config.Server, s"/metrics/$testEntityName")
+
+    val absRange = 1 to 10
+    val aggRange = 11 to 20
+
+    val metrics = new mutable.HashMap[String, MetricJSON]
+    for (i <- absRange) {
+      metrics.put(testMetricName + "-" + i, new MetricJSON(1, RecordType.ABSOLUTE.toString.toUpperCase))
+    }
+    for (i <- aggRange) {
+      metrics.put(testMetricName + "-" + i, new MetricJSON(1, RecordType.AGGREGATE.toString.toUpperCase))
+    }
+
+    val result = Await.result(
+      POST(url).setBody(
+        pretty(render(Extraction.decompose(
+          EntityJSON(
+            testPersistedDateEpoch,
+            metrics.toMap)))))
+      .apply,
+      Config.RequestTimeout
+    )
+
+    result.code.code should be (NoContent.code)
+
+    "{" +
+      (absRange.map(i =>
+        s""" "$testMetricName-$i": { "value": 1, "type": "absolute" } """
+      ) ++
+      aggRange.map(i =>
+        s""" "$testMetricName-$i": { "value": 1, "type": "aggregate" } """
+      )).mkString(", ") +
+      "}"
+  }
+
+  "Persist absolute metrics via POST" should "succeed" in {
+    persistManyMetrics(1 to 10, RecordType.ABSOLUTE)
+  }
+
+  "Persist aggregate metrics via POST" should "succeed" in {
+    persistManyMetrics(1 to 10, RecordType.AGGREGATE)
+  }
+
+  "Persist both absolute and aggregate metrics in the same request" should "succeed" in {
+    persistManyAbsoluteAndAggregate()
```

Comment:
If you expanded `persistManyMetrics` to work like:

    persistManyMetrics(1 to 10, RecordType.ABSOLUTE)
    persistManyMetrics(1 to 10, RecordType.AGGREGATE)
    persistManyMetrics(1 to 10, RecordType.ABSOLUTE, RecordType.AGGREGATE)

You wouldn't need a second method.